website/src/components/Submission/DataUploadForm.tsx

```diff
@@ -448,6 +186,170 @@ const InnerDataUploadForm = ({
+    restrictedUntil: DateTime;
+    setRestrictedUntil: (restrictedUntil: DateTime) => void;
+}) => {
+    return (
+        <div className='grid sm:grid-cols-3 gap-x-16 gap-y-4'>
+            <div>
+                <h2 className='font-medium text-lg'>Data use terms</h2>
+                <p className='text-gray-500 text-sm'>Choose how your data can be used</p>
+            </div>
+            <div className='gap-x-6 gap-y-8 col-span-2'>
+                <div className='space-y-6'>
+                    <label htmlFor='username' className='block text-sm font-medium leading-6 text-gray-900'>
+                        Terms of use for this data set
+                    </label>
+                    <div className='space-y-2'>
+                        <DataUseTermsSelector
+                            calendarUseModal
+                            initialDataUseTermsOption={dataUseTermsType}
+                            maxRestrictedUntil={dateTimeInMonths(12)}
+                            setDataUseTerms={(terms) => {
+                                setDataUseTermsType(terms.type);
+                                if (terms.type === restrictedDataUseTermsOption) {
+                                    setRestrictedUntil(DateTime.fromFormat(terms.restrictedUntil, 'yyyy-MM-dd'));
+                                }
+                            }}
+                        />
+                    </div>
+                    {dataUseTermsType === openDataUseTermsOption ? (
+                        <p className='text-sm'>Your data will be available on Pathoplexus under the open use terms.</p>
+                    ) : (
+                        <p className='text-sm'>
+                            Your data will be available on Pathoplexus, under the restricted use terms until{' '}
+                            {restrictedUntil.toFormat('yyyy-MM-dd')} and under the open use terms after that date.
+                        </p>
+                    )}
+                </div>
+            </div>
+        </div>
+    );
+};
+
+const Acknowledgement = ({
+    confirmedNoPII,
+    setConfirmedNoPII,
+    agreedToINSDCUploadTerms,
+    setAgreedToINSDCUploadTerms,
+}: {
+    confirmedNoPII: boolean;
+    setConfirmedNoPII: Dispatch<SetStateAction<boolean>>;
+    agreedToINSDCUploadTerms: boolean;
+    setAgreedToINSDCUploadTerms: Dispatch<SetStateAction<boolean>>;
+}) => {
+    return (
+        <div className='grid sm:grid-cols-3 gap-x-16 gap-y-4'>
+            <div className=''>
+                <h2 className='font-medium text-lg'>Acknowledgement</h2>
+                <p className='text-gray-500 text-sm'>Acknowledge submission terms</p>
+            </div>
+            <div className='gap-x-6 gap-y-8 col-span-2'>
+                <div>
+                    <p className='block text-sm'>
+                        Your data will be available on Pathoplexus, under the selected data use terms. Data with open
```

Comment:
Hmm.. so this feels like it has been made out of technical convenience rather than user experience

<img width="998" alt="image" src="https://github.com/user-attachments/assets/670121eb-8aa5-451b-a5c9-db97c44a748b" />

I had thought we would only remove the more specific reference to the data use terms for the revision form. IMO this is a bit of a regression and it would be better to keep the behaviour the same except for the specific case of the revision form where we don't know the terms.

Reply:
Agreed. so you're saying, keep the old behaviour for initial submit, and only have the less specific formulation on the review page? That makes sense to me.